doc/quick_start/contributor-guide.rst

```diff
@@ -0,0 +1,289 @@
+.. _contributor-guide:
+
+
+Ostro |trade| OS Contributor Guide
+##################################
+
+As an open-source project, we welcome and encourage the community to submit patches directly to the project. 
+This document explains how to participate in project conversations, log bugs and enhancement requests, 
+and submit patches to the project so your patch will be accepted quickly in the codebase. At a high-level, 
+we use the GitHub Pull Request workflow and follow the `Open Embedded guidelines for commit messages`_, 
+as explained in our `Code Review Process Guidelines`_ section below.
+
+Participating in Ostro Project Conversations
+============================================
+
+As in many open source projects, we have online options for developers and community members to converse about the Ostro Project, including
+an `Ostro-dev mailing list`_, `Ostro Project IRC channel`_ on freenode.net, `ostroproject GitHub repo`_, 
+`bug and issue tracking`_, `Ostro OS Release Notes`_, and more.  
+You can learn more about these resources and how to subscribe to and access them in :ref:`access-support`.
+
+Communication on IRC is immediate, yet transient, making it good for meetings or a quick question. 
+The mailing list is for announcements and questions for broad exposure and discussion. 
+JIRA is our issue tracking system, for feature requests and bug reports.
+
+.. _`Open Embedded guidelines for commit messages`: http://openembedded.org/wiki/Commit_Patch_Message_Guidelines
+.. _`Yocto Project`: http://yoctoproject.org
+.. _`Ostro OS Release Notes`: https://github.com/ostroproject/ostro-os/releases/
+.. _`Ostro-dev mailing list`: mailto://ostro-dev@lists.ostroproject.org
+.. _`ostroproject GitHub repo`: https://github.com/ostroproject/
+.. _`Ostro Project JIRA`: https://ostroproject.org/jira
+.. _`but and issue tracking`: https://ostroproject.org/jira
```

Comment:
fixed.  The different text was so that the words that show up in the document that you click on could be different (used in different places).